### ISSUE TITLE
Disable Google Analytics 

### ISSUE THREAD

**Issue by cb3inco** [thedevs-network/kutt#461]
It would be nice to be able to completely disable google analytics for privacy reasons. Unless I'm missing on how to hack the frontend web page.

**Comment by coppea** [comment]
yes,I want to how to hack the frontend web page. For example, modify the language and so on.
